quantmsio/core/quantms/pg.py

```
@@ -0,0 +1,787 @@
+                            "description": info["description"],
+                            "sequence_coverage": info["sequence_coverage"],
+                            "global_qvalue": info["global_qvalue"],
+                            "peptide_count": info["peptide_count"],
+                            "is_decoy": info["is_decoy"],
+                            "sequence_length": info["sequence_length"],
+                        }
+                    )
+
+            # Convert to DataFrame and load into DuckDB
+            if protein_data:
+                protein_df = pd.DataFrame(protein_data)
```

Flake8 (7.2.0)

[error] local variable 'protein_df' is assigned to but never used

(F841)